src/app/find-help/accommodation-search/accommodation-search.component.html

```diff
@@ -1,7 +1,7 @@
 <mat-card>
   <app-accommodation-search-form (search)="search($event)"></app-accommodation-search-form>
 </mat-card>
-<div class="results" *ngIf="this.total >= 0" [class.loading]="loading">
+<div class="results" *ngIf="this.total && this.total >= 0" [class.loading]="loading">
```

Comment:
Propably `*ngIf="this.total"` is enough. It would be `false` on `total === undefined, null, 0`

Reply:
Done.